src/screens/Login/Components/login-form.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Button, TextInput } from 'react-native-paper';
+import { View, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import { styles } from '../styles';
+import { Input, Text } from 'src/components';
+import { AppColor, AuthRoute } from 'src/common/enums';
+
+const LoginForm = () => {
+  const [secure, setSecure] = useState(true);
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const navigation = useNavigation();
+
+  const onLoginPressed = () => {
+    // TODO authentication logic
+  };
+
+  return (
+    <View style={styles.container}>
+      <Image
+        source={require('src/assets/images/Logo.png')}
```

Comment:
In .tsx this is different. Add this line before styles import:
```typescript
import logoImage from 'src/assets/images/Logo.png'
```
And then `source={logoImage}`

Reply:
It doesn't work for me
![image](https://user-images.githubusercontent.com/41474876/154262442-4e7ffd70-eeb3-4763-9824-6f178eb701bf.png)

I can create a separate file with exports of the images from assets.